ApiContract/request.json5

```diff
@@ -0,0 +1,41 @@
+//POST /api/advertisement
+
+//Request object
+{
+	//[Mandatory] id of the advertisement
+	"id" : "P123",
+	//[Mandatory] title of the advertisement
+	"name" : "Watch" ,
+	//[Mandatory] description of the advertisement
+	"description" : "something",
+	//[Mandatory] images of the product in the advertisement (Maximum files - 5)
```

Comment:
Max of 5 images type of restrictions should not be exposed at the contract level 
This could be handled in a different way..
But let’s discuss if this should be part of contract or not..